bin/functions

```diff
@@ -51,7 +53,7 @@ function retrieve-release-data() {
 
     # shellcheck disable=SC2046
     if [[ ! -r "${cache_file}" ]] || (( $($STAT "${STAT_OPTS[@]}" "${cache_file}") <= $(date +%s) - 3600 )) ; then
-        curl -s -f --compressed -L "https://raw.githubusercontent.com/halcyon/asdf-java/master/data/jdk-${OS}-${ARCHITECTURE}.tsv" -o "${cache_file}"
+        curl -s -f --compressed -L "https://raw.githubusercontent.com/joschi/asdf-java/aarch64-support/data/jdk-${OS}-${ARCHITECTURE}.tsv" -o "${cache_file}"
```

Comment:
This has to be reverted before merging. It only for making the CI succeed for as long there are no data files for aarch64 and armv7l in the master branch.

Commit: 784e9f7de96414dfcfe8bcdbd3d7c9a1016d78b6